.github/workflows/push_pr.yml

```diff
@@ -28,14 +28,25 @@ jobs:
   compile_and_test:
     strategy:
       matrix:
-        flink: [ 1.18-SNAPSHOT ]
-        jdk: [ '8, 11, 17' ]
+        mongodb: [ mongodb4, mongodb5, mongodb6, mongodb7 ]
+        flink: [ 1.19-SNAPSHOT, 1.20-SNAPSHOT ]
+        jdk: [ '8, 11, 17, 21' ]
```

Comment:
Will this do the same trick?

```suggestion
        mongodb: [ mongodb4, mongodb5, mongodb6, mongodb7 ]
        flink: [ 1.18-SNAPSHOT, 1.19-SNAPSHOT, 1.20-SNAPSHOT ]
        jdk: [ '8, 11, 17, 21' ]
        exclude:
          - flink: 1.18-SNAPSHOT
            jdk: 21
```

Reply:
Since only a single value can be specified in include and exclude, the jdk parameter has been changed from a sequence to a string. This allows multiple jdk versions to be conveniently declared in include, but they cannot be excluded in exclude.

Reply:
I tried the above method, and it indeed works, but it affects the structure of the tests. Additionally, version 1.18 is about to be deprecated, so I think we can maintain the current form for now.

![image](https://github.com/user-attachments/assets/6648123c-7a3d-44af-bbe8-df9c8d1eadb1)